contrib/nydusify/pkg/cache/cache.go

```diff
@@ -129,6 +129,7 @@ func (cache *Cache) recordToLayer(record *Record) (*ocispec.Descriptor, *ocispec
 		Size:      record.NydusBootstrapDesc.Size,
 		Annotations: map[string]string{
 			utils.LayerAnnotationNydusBootstrap:     "true",
+			utils.LayerAnnotationNydusFsVersion:     cache.opt.Version,
```

Comment:
Should be `cache.opt.FsVersion`, how this option be passed to the cache image?

Reply:
I checked, it is cache.opt.Version.

Reply:
```
// Opt configures Nydus cache
type Opt struct {
	// Maximum records(bootstrap layer + blob layer) in cache image.
	MaxRecords uint
	Version    string
	// Make cache image manifest compatible with the docker v2 media
	// type defined in github.com/containerd/containerd/images.
	DockerV2Format bool
	// The blob layer record will not be written to cache image if
	// the backend be specified, because the blob layer will be uploaded
	// to backend.
	Backend backend.Backend
}
```

Reply:
@yawqi  This `Version` field is used to indicate the version of cache image (not fs version), nydusify will use cache records depending on the version. :)

Reply:
We can add some comments to distinguish these two fields:

``` go
// Opt configures Nydus cache
type Opt struct {
	...
	// For the version of cache image, we need to discard cache layers
	// when the required version (specified by `--build-cache-version`)
	// is unmatched with the cache image version, for example nydus
	// bootstrap format has a minor upgrade.
	Version    string
	// Bootstrap's RAFS version of cache image, we need to discard cache
	// layers when the required version (specified by `--fs-version`) is
	// unmatched with the fs version recorded in cache image, for example
	// we can't use rafs v5 cache layers for rafs v6 image.
	FsVersion string
	// Make cache image manifest compatible with the docker v2 media
	// type defined in github.com/containerd/containerd/images.
	...
}
```

Reply:
Done.